### ISSUE TITLE
Support staff lines without a starting bar line

### ISSUE THREAD

**Issue by sonovice** [sonovice/smude#3]
Right now the vanishing point computation (and some other things) depend on staff lines starting and ending with a bar line. Especially single instrument staves usually do not have such a starting line, resulting in various errors.

Basic idea:
1. Do staff line detection as already implemented
2. Draw blobs at the start and end of detected staff lines
3. Use Hough Transform on these "blob images" do detect the "non-existent bar lines"
4. Profit.

**Comment by rettinghaus** [comment]
I get a 

> ValueError: Staff lines could not be detected!

although they are definetly there.

**Comment by sonovice** [comment]
Yes, the entire codebase was more of a research prototype, not production ready software, unfortunately. I will ask some LLM to implement the idea from the top here, maybe it'll help.

Your problem seems to be more about staff lines, not system bar lines, though. Could you upload an example image?
